frontend/src/components/ViewActivities/activityday.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Accordion from 'react-bootstrap/Accordion';
+import Card from 'react-bootstrap/Card';
+import Activity from './activity';
+import * as activityFns from './activityfns';
+import * as utils from '../Utils/utils.js'
+
+class ActivityDay extends React.Component {
+  /** @inheritdoc */
+  render() {
+    const sortedActivities = Array.from(this.props.activities)
+          .sort(activityFns.compareActivities);
```

Comment:
Nit: declare completely on separate line like so.
```suggestion
    const sortedActivities = 
        Array.from(this.props.activities).sort(activityFns.compareActivities);
```